extensions/binding/org.eclipse.smarthome.binding.weatherunderground/OSGI-INF/WeatherUndergroundDiscoveryService.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shouldn't we go for annotated DS now?
@htreu You had updated the archetype accordingly, but we haven't mentioned anything in the coding guidelines yet. Time for it, do you agree?